waffle-compiler/test/flattener/flattener.ts

```diff
@@ -0,0 +1,34 @@
+import {expect} from 'chai';
+import fs from 'fs-extra';
+import {inputToConfig} from '../../src/config';
+import {flattenAndSave} from '../../src/flattener';
+
+const sourceDirectory = './test/flattener/testSource';
+const flattenOutputDirectory = './test/flattener/flattenFiles';
+const compilerVersion = 'v0.5.9+commit.e560f70d';
+const config = inputToConfig({
+  sourceDirectory,
+  flattenOutputDirectory,
+  compilerVersion
+});
+
+describe('flattening', () => {
+  after(async () => {
+    await fs.remove(flattenOutputDirectory);
+  });
+
+  it('should include the parent if the child file', async () => {
```

Comment:
```suggestion
  it('should include the parent in the child file', async () => {
```

Reply:
Also, this test doesn't check much. Why not create the expected file by hand and compare whole file contents?